src/Assets/ugui-mvvm/PropertyBinding.cs

```diff
@@ -309,14 +312,22 @@ public void ClearHandlers()
         ComponentPath _source;
         public ComponentPath Source => _source;
 
+        [SerializeField]
+        private BindingUpdateTrigger _sourceUpdateTrigger = BindingUpdateTrigger.None;
+
+#pragma warning disable 0414
```

Comment:
>0414 [](start = 24, length = 4)

Add a comment explaining which warning this is, and optionally why it was needed.